src/main/java/Group15/ViewController.java

```diff
@@ -1,16 +1,25 @@
 package Group15;
 
+import Group15.Api.BodyPart;
+import Group15.Api.Api;
+import javafx.geometry.Insets;
 import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ToggleButton;
+import javafx.scene.layout.VBox;
 import javafx.stage.Stage;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class ViewController
 {
     private static Stage stage;
 
-    public static void init()
+    public static void init(Stage stage)
     {
-        ViewController.stage = new Stage();
-        ViewController.stage.show();
+        stage.setScene(createMuscleSelectorScene());
```

Comment:
What happens the next time someone tries to use the setScene or getScene method? :)